src/coreclr/vm/mono/mono_coreclr.cpp

```diff
@@ -69,26 +71,29 @@ typedef intptr_t ManagedStringPtr_t;
 // Generated by UnityEmbedHost.Generator - Commit these changes
 struct HostStruct
 {
+    int32_t (*add_system_assembly_search_path)(const char* raw_path);
```

Comment:
Do we really need that ? We will implement custom assembly load context sub-classes in the editor loader. They can handle assembly resolution

Reply:
this is needed to avoid system assemblies loading in the subsequent ALCs - that makes the startup quite faster
since the ALC manager is currently in the embed-host I pass the .net folder via HostStruct
note that the HostStruct is internal to the embed-host and not exposed, the path is inferred from monoPaths we pass to the mono_jit_init

Reply:
This is actually what we want: ultimately, the default ALC should only hold PrivateCorlib + UnityEditor.Loader